plugin/src/main/java/org/wildfly/plugins/bootablejar/maven/goals/DevBootableJarMojo.java

```diff
@@ -74,6 +74,9 @@ public void execute() throws MojoExecutionException, MojoFailureException {
         hollowJar = true;
         super.execute();
 
+        // Always disable color when printing to file.
+        arguments.add("-Dorg.jboss.logmanager.nocolor=true");
```

Comment:
We may want to to add this in the builder with `builder.addJavaOption("-Dorg.jboss.logmanager.nocolor=true")` for random cases where a user may want to override the property. Something like:
```
final BootableJarCommandBuilder commandBuilder = BootableJarCommandBuilder.of(Utils.getBootableJarPath(null, project, "dev"))
        .addJavaOption("-Dorg.jboss.logmanager.nocolor=true")
        .addJavaOptions(jvmArguments)
        .addServerArguments(arguments);

```

Note adding it the JVM options sets it early so the log manager will see it. Adding it before the `jvmArguments` are set allows the `jvmArguments` to override it.

Reply:
Thank-you @jamezp , I made the change.